src/components/molecules/ToolCard.tsx

```
@@ -0,0 +1,270 @@
+                                onClick={() => setShowExecuted(!showExecuted)}>
+                                {showExecuted ?
+                                    <ChevronDown /> :
+                                    <ChevronRight />
+                                }
+                                <span>実行済みのツール（{executedTools}）</span>
+                            </button>}
+                        {showExecuted && (
+                            cmds.filter(cmd => cmd.result !== null).reverse().map(({ call_id, args, name, method, description, result }) => {
+                                return (
+                                    <div className="cursor-default h-full py-3 px-6 border border-neutral-300 rounded-md shadow mb-2">
+                                        <h3 className="flex items-center justify-between text-xl font-semibold text-neutral-700 border-b pb-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)